src/content/docs/Language Fundamentals/comments.md

```diff
@@ -46,12 +46,12 @@ The following was extracted:
 |---------------|----------------------------------------------------:|
 | `@param     ` |            `@param [<ref>] <param> [<description>]` |
```

Comment:
The `:` is now mandatory in front of the description, so it's `[ : <description>]`